### PR TITLE
Feature/cvsb 14890

### PR DESCRIPTION
**Pull request by rspasici**
PR for : https://jira.dvsacloud.uk/browse/CVSB-14890
Added examiner notes field

### CODE REVIEW

File: src/app/technical-record/adr/additional-adr-details/additional-adr-details.component.spec.ts [hunk 1]
```diff
@@ -25,7 +25,10 @@ describe('AdditionalAdrDetailsComponent', () => {
 
   it('should create view only with populated data', () => {
     component.edit = false;
-    component.adrDetails = TESTING_UTILS.mockAdrDetails();
+    component.adrDetails = TESTING_UTILS.mockAdrDetails({
+      additionalExaminerNotes: 'examination note',
+      adrCertificateNotes: ' certificate note '
+    });
```
Comment (alleypa) [inline src/app/technical-record/adr/additional-adr-details/additional-adr-details.component.spec.ts:31]: move this inside the beforeEach

File: src/app/technical-record/adr/additional-adr-details/additional-adr-details.component.spec.ts [hunk 2]
```diff
@@ -34,7 +37,10 @@ describe('AdditionalAdrDetailsComponent', () => {
 
   it('should render the editable component if edit is true', () => {
     component.edit = true;
-    component.adrDetails = { additionalExaminerNotes: 'some notes!!!' } as AdrDetails;
+    component.adrDetails = {
+      additionalExaminerNotes: 'some notes!!!',
+      adrCertificateNotes: 'some certificate notes!'
+    } as AdrDetails;
```
Comment (alleypa) [inline src/app/technical-record/adr/additional-adr-details/additional-adr-details.component.spec.ts:43]: move this from here to inside the beforeEach above

**Comment by JonLazariniDD** [comment]
Let's please add a description and link to the PR.

**Comment by m-ujaffer** [comment]
Please don't merge till further notice.

**Pull request merged**